google-storage/src/googleUtility.ts

```diff
@@ -0,0 +1,61 @@
+// import service account details, bucketname, filename and other details from your configuration file
+import {
+	serviceAccount,
+	baseUrl,
+	bucketName,
+} from '../config';
+/**
+ * Check validity of Service Account configuration
+ * @param config
+ * @returns {{private_key}|{client_email}|{project_id}|any}
+ */
+
+const checkServiceAccount = () => {
```

Comment:
Change the function interface to take `serviceAccount`, `bucketName`, and so on as parameters.

This function should work without these values having to be set as globals in the users program.